### ISSUE TITLE
Every release newer than 9.1.238837 stops working with yabai

### ISSUE THREAD

**Issue by hpfmn** [overtake/TelegramSwift#873]
For some reasons version newer than 9.1.238837 stop working with the tiling window manager yabai

see also https://github.com/koekeishiya/yabai/issues/957

**Comment by zakroma** [comment]
Same problem over here :(

**Comment by kalamov** [comment]
Temporarily installed manually the old version 9.1 (238837), link - https://install.appcenter.ms/users/keepcoder/apps/telegram-swift/distribution_groups/public

**Comment by zakroma** [comment]
Great news everyone!

Telegram 9.1.239550 fixes the problem and is getting recognized by yabai!

**Comment by overtake** [comment]
That is good. Will be ok in next release then

**Comment by zakroma** [comment]
> @zakroma Could you please point me to the fix - curious why Telegram stopped exposing its accessibility elements.

IDK what exactly they have fixed, but you can get the latest version (the one I mentioned above) [here](https://install.appcenter.ms/users/keepcoder/apps/telegram-swift/distribution_groups/public)

**Comment by iSapozhnik** [comment]
Yeah, I mainly was curious about the technical side of the question since I'm working on an app (window manager) that relies on accessibility stuff, and while testing my app I also figured out that it does not really work with Telegram. That's how I found this thread :)

**Comment by hpfmn** [comment]
Yeah it is really confusing to map the releases to specific commits. Are there non public branches for the current betas?

**Comment by godbout** [comment]
versions newer than 9.1.238837 break my apps kindaVim and Wooshy. the AXCells are gone (they should be there within the AXRows) and it's not even possible anymore to do a basic ping of the input. the macOS AX returns the -25212 AXError which means that the input as `noValue`. basically, it just broke.
